config.py

```diff
@@ -10,6 +10,8 @@ class BaseConfig(object):
     SQLALCHEMY_DATABASE_URI = 'sqlite:///:memory:'
     SQLALCHEMY_TRACK_MODIFICATIONS = False
 
+    UNVERIFIED_USER_THRESHOLD = 1814400  # 21 days
```

Comment:
After asking mentors we decide to make it 30 days, sorry :)

Reply:
Okay, will change :)